code/game/objects/items/devices/inhaler.dm

```diff
@@ -11,10 +11,26 @@
 	throw_range = 20
 	force = 0
 	var/cooldown = 0
-	
+
 /obj/item/device/inhaler/attack_self(mob/user)
 	if(cooldown < world.time - 8  && ishuman(user))
-		var/mob/living/carbon/human/H = user
-		playsound(user, 'sound/effects/spray2.ogg', 20, 1)
-		H.reagents.add_reagent(ALBUTEROL, 5)
-		cooldown = world.time
+		puff(user)
+
+/obj/item/device/inhaler/attack(mob/living/M, mob/user)
+	if(!ishuman(M) || user.zone_sel.selecting != "mouth" || user.a_intent != I_HELP)
```

Comment:
Didn't we coerce someone to make defines for zones at some point?

Reply:
On second thought, is there anything to gain from forcing the player to select the mouth? What else are you going to do to someone else with an inhaler? Smack them with it?